homeassistant/components/zwave_js/config_flow.py

```diff
@@ -346,11 +346,13 @@ class ZWaveJSConfigFlow(BaseZwaveJSFlow, ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    _title: str
+    unique_id: str
```

Comment:
I don't think this is correct. We even have logic to check if the unique_id is falsy since it may not be set until at a later stage in the flow.